modules/troubleshooting-network-observability-lokistack-rate-limits.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="nw-lokistack-rate-limits_{context}"]
+= Configuring LokiStack Rate Limits
+
+Two notable LokiStack specifications, `ingestionRate` and `ingestionBurstSize`, might have an impact on the flow processing pipeline. If these rate limits are reached, Loki returns an error when `flowlogs-pipeline` tries to send batches. You can disable the maximum number of active streams per user and increase the number of maximum global streams per user to `5000`.
+
+.Procedure
+
+. Run the following command to get the `lokistack-config` configmap in the `netobserv` namespace:
++
+[source,terminal]
+----
+$ oc get configmap lokistack-config -n netobserv -o yaml | yq '.binaryData | map_values(. | @base64d)' > binaryData.txt
+----
+
+. Edit the following parameters in the `binaryData.txt` file:
++
+[source,yaml]
+----
+spec:
+  limits_config:
+    max_streams_per_user: 0
+    max_global_streams_per_user: 5000
+----
```

Comment:
As you mention `ingestionRate` and `ingestionBurstSize` in the previous paragraph, are you trying to cover [Ingestion rate limit exceeded](https://github.com/netobserv/documents/blob/main/loki_config.md#ingestion-rate-limit-exceeded) here ?

It's different from [Maximum active stream limit exceeded](https://github.com/netobserv/documents/blob/main/loki_config.md#maximum-active-stream-limit-exceeded) about `max_streams_per_user` and `max_global_streams_per_user`

I guess you should have two separate files for this two cases

Reply:
Addressed